scripts/qrzgrid.tcl

```diff
@@ -2129,5 +2129,21 @@ proc msg_spot {nick uhand handle input} {
 	close $fd
 }
 
+bind pub - !masters masters
+bind pub - !themasters masters
+set mastersbin "/home/eggdrop/bin/masters"
+proc fieldday { nick host hand chan text } {
```

Comment:
need to change this proc name, or else it will conflict with the `!fieldday` command